[2021-12-21 23:06:55,264] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline.move_file scheduled__2021-12-19T00:09:00+00:00 [queued]>
[2021-12-21 23:06:55,269] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline.move_file scheduled__2021-12-19T00:09:00+00:00 [queued]>
[2021-12-21 23:06:55,269] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 23:06:55,269] {taskinstance.py:1242} INFO - Starting attempt 4 of 4
[2021-12-21 23:06:55,269] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 23:06:55,278] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): move_file> on 2021-12-19 00:09:00+00:00
[2021-12-21 23:06:55,279] {standard_task_runner.py:52} INFO - Started process 104356 to run task
[2021-12-21 23:06:55,281] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'move_file', 'scheduled__2021-12-19T00:09:00+00:00', '--job-id', '185', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmp7a2b0ylc', '--error-file', '/tmp/tmp_korm6vs']
[2021-12-21 23:06:55,281] {standard_task_runner.py:77} INFO - Job 185: Subtask move_file
[2021-12-21 23:06:55,299] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline.move_file scheduled__2021-12-19T00:09:00+00:00 [running]> on host myamya
[2021-12-21 23:06:55,316] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline
AIRFLOW_CTX_TASK_ID=move_file
AIRFLOW_CTX_EXECUTION_DATE=2021-12-19T00:09:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-19T00:09:00+00:00
[2021-12-21 23:06:55,317] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-21 23:06:55,317] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cp /home/myamya/project/image_files/cropped_512_image/* /home/myamya/project/images/real/ | cp /home/myamya/project/image_files/sketched2/* /home/myamya/project/images/sketch/']
[2021-12-21 23:06:55,321] {subprocess.py:85} INFO - Output:
[2021-12-21 23:06:55,322] {subprocess.py:89} INFO - cp: 일반 파일 '/home/myamya/project/images/real/'을(를) 생성할 수 없음: 디렉터리가 아닙니다
[2021-12-21 23:06:55,322] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-21 23:06:55,334] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=move_file, execution_date=20211219T000900, start_date=20211221T140655, end_date=20211221T140655
[2021-12-21 23:06:55,373] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-21 23:06:55,402] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
